### DISCUSSION TITLE
How to change length of text or make it wrap for action help text?

### DISCUSSION THREAD

**Discussion by neonvoidx** [ibhagwan/fzf-lua#1579] [Q&A]
![image](https://github.com/user-attachments/assets/9abbef69-713d-4b66-995c-fd92d5864815)
I clearly cant read the action text, can we make this wrap somehow

**Comment by ibhagwan** [comment]
> I clearly cant read the action text, can we make this wrap somehow

You can hide the preview with F4 or rotate with F5/6.

Regarding wrapping, this can be enabled globally for the window with `ctrl-/` or `alt-/`, see #1565 on how to customize it or get more ideas.

**Reply by neonvoidx** [reply]
F4 helps thanks, wish the text could wrap but oh well.

**Reply by ibhagwan** [reply]
> F4 helps thanks, wish the text could wrap but oh well.

Read the linked issue, the text can wrap by default with `fzf_opts = { ["--wrap"] == true }`.

**Reply by neonvoidx** [reply]
this doesn't seem to affect the help text for actions?

**Reply by neonvoidx** [reply]
![image](https://github.com/user-attachments/assets/18f20636-5af4-4f2c-b834-65149a2754f9)

**Reply by neonvoidx** [reply]
Heres another example
![image](https://github.com/user-attachments/assets/0ea7094d-f728-49b6-b26b-8986246d9872)

you can see the results being wrapped but not the action help text

**Reply by ibhagwan** [reply]
> this doesn't seem to affect the help text for actions?

In this case you'd need to submit a feature request [upstream](https://github.com/junegunn/fzf) with the subject "header lines unaffected by `--wrap`" as this is handled by fzf not fzf-lua.

Alternatively fzf-lua could set 2 header lines depending on available lenght but that is overly complicated in what IMHO should be handled by fzf  (fzf-lua will have to take into account "native" previewers such as `bat` and calculate space based on horizontal/vertical layout).
